dlrover/python/elastic_agent/torch/training.py

```diff
@@ -642,16 +648,16 @@ def _invoke_run(self, role: str = DEFAULT_ROLE) -> RunResult:
                 return run_result
             elif state in {WorkerState.UNHEALTHY, WorkerState.FAILED}:
                 logger.error(f"The worker fails with {run_result.failures}")
-                self._report_failure_to_master(run_result.failures)
```

Comment:
we need a  faults detection here，not just depend on training process status.

Reply:
We plan to add this in future pr.